Automattic/vip-go-mu-plugins
# FORNO-283: Increase timeout for file deletion

## Description

Increase the timeout for file delete requests to the File Service to reduce the amount of PHP warnings occurring due to request timeout errors.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Upload changes to a sandboxed site
1. Go to `wp-admin` > `media`
1. Ensure that media upload and delete is working as expected
